+hw/DaqController.m

```diff
@@ -223,26 +284,31 @@ function queue(obj, names, waveforms)
       len = cellfun(@numel, waveforms);
       defaultValues = [obj.SignalGenerators.DefaultValue];
       samples = repmat(defaultValues(obj.AnalogueChannelsIdx), max(len), 1);
-      for ii = 1:numel(waveforms)
-        cidx = strcmp(names{ii}, obj.ChannelNames);
-        assert(sum(cidx) == 1, 'Channel name mismatch');
```

Comment:
This assert made sure there weren't two channels of the same name.  Shouldn't we keep it in?

Reply:
technically you can have two channels of same name if they have different signal generators, channel ids, etc...